lib/crons.ex

```diff
@@ -0,0 +1,131 @@
+defmodule ApplicationRunner.Crons do
+  @moduledoc """
+    ApplicationRunner.Crons delegates methods to the corresponding service.
+  """
+
+  import Ecto.Query, only: [from: 2, from: 1]
+
+  alias ApplicationRunner.Crons.Cron
+  alias ApplicationRunner.Environment
+  alias ApplicationRunner.Errors.{BusinessError, TechnicalError}
+  alias ApplicationRunner.EventHandler
+  alias ApplicationRunner.Repo
+  alias Crontab.CronExpression.{Composer, Parser}
+
+  def run_env_cron(
+        action,
+        props,
+        event,
+        env_id
+      ) do
+    with {:ok, metadata} <-
+           Environment.create_metadata(env_id),
```

Comment:
REFACTO: here create the metadata with fun params, call AppSoket.do_create_env_token for the token